components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/UserManagementService.java

```diff
@@ -409,6 +409,40 @@ Response getUsers(
     Response getUserCount();
 
     @GET
+    @Path("/checkUser")
```

Comment:
Please adhere the [WSO2 API Guideline section 5.1](http://wso2.com/whitepapers/wso2-rest-apis-design-guidelines/#51) when designing resource URIs. Should avoid using camelCases.